discovery/spec.md

```diff
@@ -114,6 +90,7 @@ defined:
 - Description: A unique identifier for the resource.
```

Comment:
Unique within what scope?

Reply:
Is it ok that it says:
```
MUST be unique across all instances of the this resource type within the current Discovery Service.
```
down below?

Reply:
Do we have a definition for the identity of the Discovery Service?

Reply:
@deissnerk or did you want me to repeat it in the Description line?

Reply:
@sasha-tkachev what do you mean by "definition for the identity" ? Below we say it must conform to RFC3986/3.3. Or are you asking for the semantic meaning behind the string? If so, I'm not sure we can get into that.